website/content/en/pre-docs/tasks/running-pods.md

```diff
@@ -0,0 +1,240 @@
+
+The `IgnoredDuringExecution` part of each tells the pod to keep running, even if conditions change on the node so the rules no longer matched.
+You can think of these concepts as `required` and `preferred`, since Kubernetes never implemented other variants of these rules.
+
+All examples below assume that the provisioner doesn't have constraints to prevent those zones from being used.
+The first constraint says you could use `us-west-2a` or `us-west-2b`, the second constraint makes it so only `us-west-2b` can be used.
+
+```
+ affinity:
+   nodeAffinity:
+     requiredDuringSchedulingIgnoredDuringExecution:
+       nodeSelectorTerms:
+         - matchExpressions:
+           - key: "topology.kubernetes.io/zone"
+             operator: "In"
+             values: ["us-west-2a, us-west-2b"]
+           - key: "topology.kubernetes.io/zone"
+             operator: "In"
+             values: ["us-west-2b"]
+```
+
+Changing the second operator to `NotIn` would allow the pod to run in `us-west-2a` only:
+
+```
+           - key: "topology.kubernetes.io/zone"
+             operator: "In"
+             values: ["us-west-2a, us-west-2b"]
+           - key: "topology.kubernetes.io/zone"
+             operator: "NotIn"
+             values: ["us-west-2b"]
+```
+
+Continuing to add to the example, `nodeAffinity` lets you define terms so if one term doesn't work it goes to the next one.
+Here, if `us-west-2a` is not available, the pod can go to the East zone and run on a spot instance (notice the AWS-specific key).
+
+
+```
+ affinity:
+   nodeAffinity:
+     requiredDuringSchedulingIgnoredDuringExecution:
+       nodeSelectorTerms:
+         - matchExpressions: # OR
+           - key: "topology.kubernetes.io/zone" # AND
+             operator: "In"
+             values: ["us-west-2a, us-west-2b"]
+           - key: "topology.kubernetes.io/zone" # AND
+             operator: "NotIn"
+             values: ["us-west-2b"]
+         - matchExpressions: # OR
+           - key: "karpenter.sh/capacity-type" # AND
+             operator: "In"
+             values: ["spot"]
+           - key: "topology.kubernetes.io/zone" # AND
+             operator: "In"
+             values: ["us-west-2d"]
+```
+In general, Karpenter will go through each of the `nodeSelectorTerms` in order and take the first one that works.
+However, if Karpenter fails to provision on the first `nodeSelectorTerms`, it will try again using the second one.
+If they all fail, Karpenter will fail to provision the pod.
+Karpenter will backoff and retry over time.
+So if capacity becomes available, it will schedule the pod without user intervention.
+
+## Taints and tolerations
+
+Taints are the opposite of affinity.
+Setting a taint on a node tells the scheduler to not run a pod on it unless the pod has explicitly said it can tolerate that taint.
+This example shows a Provisioner that was set up with a taint for only running pods that require a graphics processing unit, such as the following:
```

Comment:
Graphics processing unit is a hilariously outdated name these days, since this is used for all sorts of parallel computing use cases. 
```suggestion
This example shows a Provisioner that was set up with a taint for only running pods that require a GPU, such as the following:
```

Reply:
Done.